pkgs/development/python-modules/hypothesis.nix

```diff
@@ -21,7 +21,7 @@ buildPythonPackage rec {
     sha256 = "1s911pd3y9hvk0hq2fr6i68dqv1ciagryhgp13wgyfqh8hz8j6zv";
   };
 
-  checkInputs = stdenv.lib.optionals doCheck [ pytest flake8 flaky ];
+  checkInputs = stdenv.lib.optionals doCheck [ pytest pytest_xdist flake8 flaky ];
```

Comment:
Fails to build without it for me. They have a `tox.ini` that adds `-n 2` in its `addopts=`, which is an option only understood by xdist.